menuinst/_schema.py

```diff
@@ -27,7 +27,8 @@
 
 log = getLogger(__name__)
 SCHEMA_DIALECT = "http://json-schema.org/draft-07/schema#"
-SCHEMA_VERSION = "1"
+# We follow schemaver
+SCHEMA_VERSION = "1-1-0"
```

Comment:
Should we always use the latest version here, similar to what you're doing in `docs/source/conf.py`?

Reply:
We'd run into a catch-22, because to bump it we'd need to first copy the latest file with a new name, and that feels clunky and hard to track via diffs and code reviews.

We could import from `menuinst.platforms.base` though, but I didn't do that because (1) we would start depending on the project itself, and so far this module was standalone, and (2) this feels like the natural place to define the schema version. That's why I "tied" these hardcoded values via a test instead.

What do you think? Would you consider this a blocker?